pyresample/kd_tree.py

```diff
@@ -926,11 +930,11 @@ def query_no_distance(target_lons, target_lats, valid_output_index,
     if index_array.ndim == 1:
         index_array = index_array[:, None]
 
-    # KDTree query returns out-of-bounds neighbors as `len(arr)`
+    # KDTree query returns out-of-bounds neighbours as `len(arr)`
```

Comment:
@pnuu We are trying to stick with US English spelling of things now. Your change makes this the UK English. Please undo these changes.

Reply:
That will break backwards compatibility, `neighbours` is used in the method and function names.

Reply:
True, but not sure comments needed to be changed. I'd be willing to deprecate the old neighbours methods over time (like until a 2.0 release).